packages/pwa-buildpack/package.json

```diff
@@ -46,6 +46,7 @@
     "npm-run-all": "^4.1.2",
     "react": "^16.1.1",
     "rimraf": "^2.6.2",
+    "strip-ansi": "^4.0.0",
```

Comment:
This should be in `dependencies`

Reply:
I might be missing something; this PR only uses `strip-ansi` directly in a test. Seems like that's a devdep.